torch/utils/_sympy/functions.py

```diff
@@ -310,3 +310,10 @@ def eval(cls, *args):
                 [int(a) for a in stride_args]
             )
         return None
+
+
+class Round(sympy.Function):
```

Comment:
I think we should probably split this into `RoundInt` and `RoundFloat` or similar, given that they return different types. This also would resolve the "sympy doesn't allow None" issue. It also would mean we could add the `RoundInt.is_integer = True` assumption.

Reply:
Sure, makes sense to me.